service/d-installer.gemspec

```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.5.0"
 
+  spec.add_development_dependency "byebug"
```

Comment:
well, I am not sure if we should add byebug to devel dependencies because debug tool is developer preference and also for TW there are already debug gem that can replace byebug and has some additional features. rake and packaging tasks is something that needs everyone who use rake there. But debugger is tool that developer use.

Reply:
Maybe we need a development enviroment or something similar, because using the byebug package from the system does not work. AFAIS, *bundle exec* only uses gems from the gemspec.